### PR TITLE
CTECH-559: adding TCP param

### PR DESCRIPTION
**Pull request by slemasne**

### CODE REVIEW

**Review by rizwansaeed — APPROVED** [review]

File: .github/workflows/build-and-test.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# This job runs the project tests
+
+name: Build and test
+
+# Trigger the workflow on push or pull request to master or develop
+
+on:
+  push:
+    branches: [ develop ]
```
Comment (rizwansaeed) [inline .github/workflows/build-and-test.yaml:9]: i don't think we need the `develop` branch

File: .github/workflows/build-and-test.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          FBN_USERNAME: ${{ secrets.MASTER_FBN_USERNAME }}
+          FBN_PASSWORD: ${{ secrets.MASTER_FBN_PASSWORD }}
+          FBN_CLIENT_ID: ${{ secrets.MASTER_FBN_CLIENT_ID }}
+          FBN_CLIENT_SECRET: ${{ secrets.MASTER_FBN_CLIENT_SECRET }}
+          FBN_LUSID_API_URL: ${{ secrets.MASTER_FBN_LUSID_API_URL }}
+          FBN_DRIVE_API_URL: ${{ secrets.MASTER_FBN_DRIVE_API_URL }}
+          FBN_APP_NAME: ${{ secrets.MASTER_FBN_CLIENT_ID }}
+          FBN_LUSID_ACCESS_TOKEN: ${{ secrets.MASTER_FBN_ACCESS_TOKEN }}
+        run: | 
+          echo "env variables for DEVELOP have been set"
```
Comment (rizwansaeed) [inline .github/workflows/build-and-test.yaml:57]: should be `MASTER`?

File: .github/workflows/build-and-test.yaml (hunk 1 above)
Comment (slemasne) [reply .github/workflows/build-and-test.yaml]: Agreed. Updated.

File: .github/workflows/build-and-test.yaml (hunk 2 above)
Comment (slemasne) [reply .github/workflows/build-and-test.yaml]: Agreed. Updated.

**Pull request merged**